packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -87,6 +87,16 @@ const MicOff = styled(Icons.MicOff).attrs({
   height: 24,
 })``;
 
+const Video = styled(Icons.Camera).attrs({
+  width: 24,
+  height: 24,
+})``;
+
+const VideoOff = styled(Icons.CameraOff).attrs({
+  width: 24,
+  height: 24,
+})``;
```

Comment:
Nitpic. Could we do this?
```suggestion
const videoIconSizes = {
  width: 24,
  height: 24,
};
const Video = styled(Icons.Camera).attrs(videoIconSizes)``;
const VideoOff = styled(Icons.CameraOff).attrs(videoIconSizes)``;
```

Reply:
Did it for all of them 👍🏻  https://github.com/GoogleForCreators/web-stories-wp/pull/11887/commits/a466dba336076c205a769fc227c8c3eb0d8455ef